pages/products.html

```diff
@@ -0,0 +1,308 @@
+                    <input id="checkbox-unchecked" class="form-check-input" type="checkbox"/> 
+                    <label for="checkbox-unchecked">Kitchen & appliances</label>
+                </li>
+                <li>
+                    <input id="checkbox-unchecked" class="form-check-input" type="checkbox"/> 
+                    <label for="checkbox-unchecked">Beds & mattresses</label>
+                </li>
+                <li>
+                    <input id="checkbox-unchecked" class="form-check-input" type="checkbox"/> 
+                    <label for="checkbox-unchecked">Decoration</label>
+                </li>
+            </ul>
+         </div>
+         <div class="mb-2">
+            <h4 class="pb-1">Rating</h4>
+            <ul class="list-style-none">
+                <li>
+                    <input id="radio-checked" class="form-check-input" type="radio" checked/>           
+                    <label for="radio-checked">4 stars & above</label>
+          
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
+                    <label for="radio-unchecked">3 stars & above</label>
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
+                    <label for="radio-unchecked">2 stars & above</label>
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
+                    <label for="radio-unchecked">1 stars & above</label>
+                </li>
+            </ul>
+         </div>
+         <div class="mb-2">
+            <h4 class="pb-1">Sort by</h4>
+            <ul class="list-style-none">
+                <li>
+                    <input id="radio-checked" class="form-check-input" type="radio" checked/>           
+                    <label for="radio-checked">Price:Low to High</label>
+          
+                </li>
+                <li>
+                    <input id="radio-unchecked" class="form-check-input" type="radio"/>
+                    <label for="radio-unchecked">Price:High to Low</label>
+                </li>
+            </ul>
+         </div>
+     </div>
+     <div class="m-2 products-grid-container">
+         <h3 class="pb-1 pl-2 ">Products</h3>
+         <div class="products-container d-flex align-items-stretch">
+         <div class="card flex-column card-vert card-shadow">
+            <span class="badge add-to-fav" title="Add to favourite">
+              <span class="material-icons icon-red"> favorite </span>
+            </span>
+            <img
+              src="https://img.freepik.com/free-photo/mid-century-modern-living-room-interior-design-with-monstera-tree_53876-129803.jpg?w=740"
+              class="card-img-vert"
+            />
+            <div class="card-body">
+              <a class="card-title" href="#card">Strandmon</a>
```

Comment:
what does the value `#card` for the href attribute mean here?

Reply:
Its from component library. so, it will point out to card component. I removed it and changed it to # for Ecomm now